packages/pn-commons/src/components/__test__/CustomTooltip.test.tsx

```diff
@@ -54,10 +54,10 @@ describe('CustomTooltip Component', () => {
     // render component
     const { container, getByRole } = render(
       <CustomTooltip tooltipContent="Mocked content" openOnClick={true}>
-        <p>Mocked Text</p>
-      </CustomTooltip>
+        <button onClick={true ? ()=>{}: undefined}>Mocked Text</button>     
```

Comment:
Why this change?

Reply:
by entering the condition I had to insert a react element or an interactable element to insert the onclick